detect_secrets/pre_commit_hook.py

```diff
@@ -144,7 +146,7 @@ def pretty_print_diagnostics(secrets: SecretsCollection, width: int = 80) -> Non
     )
     for suggestion in [
         'For information about putting your secrets in a safer place, '
-        f'please ask {os.environ.get("DETECT_SECRETS_SECURITY_TEAM", "in #security")}',
+        f'please ask {os.environ.get("DETECT_SECRETS_SECURITY_TEAM")} in {SECURITY_SLACK_CHANNEL}',
```

Comment:
The bad news is that this isn't going to work.

```python
>>> print(f'team: {os.environ.get("blah")}')
team: None
```

The good news is that it looks like this PR isn't needed? That is, you can just do:

```bash
DETECT_SECRETS_SECURITY_TEAM='in #blah' detect-secrets scan ...
```

and this code will already do what you're looking for it to do:

```
For information about putting your secrets in a safer place, please ask in #blah.
```